feature-libs/asm/customer-360/components/asm-customer-360/asm-customer-360.component.ts

```diff
@@ -143,10 +140,6 @@ export class AsmCustomer360Component
     this.setTabData();
   }
 
-  ngAfterViewInit(): void {
-    this.updateTabIndex(this.activeTab);
```

Comment:
Is this not needed anymore?

Reply:
we don't need this anymore because we want to focus on the close button when the dialog opens.